src/docs/santiment-queries/bridges/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Bridge Transactions Table Overview
+author: Santiment Team
+date: 2024-02-21
+description: Discover how Santiment Queries gather data on cryptocurrency exchanges across various blockchain networks.
+---
+
+## Overview
+
+A crypto bridge is a platform or protocol that facilitates the exchange of cryptocurrencies between different blockchain networks, promoting interoperability and liquidity within the cryptocurrency ecosystem.
+
+In **Santiment Queries**, we collect data from platforms or protocols which are trading cryptocurrencies across diverse blockchain networks:
+
+- Across
+- Arbitrum_bridge
+- Manta_pacific
+- Multichain
+- Op_bridge
+- Polygon_bridge
+- Squid
+- Stargate
+- Wbtc
+- Zksync_era_bridge
+- Zksync_lite
+
+## Usage
+
+Data Analysis and Reporting: This repository facilitates the analysis and reporting of transactions across various types of bridges, offering valuable insights into their utilization and acceptance.
+
+Monitoring and Alert Systems: Utilizing transaction data from bridges, teams can establish monitoring and alert systems to detect irregular or questionable activities across diverse bridge networks.
+
+Integration with Santiment Products: The exported data can seamlessly integrate with other Santiment products, like Sanbase, enhancing the overall understanding of the cryptocurrency landscape.
+
+## Exploring the Tables
+
+We can inspect the table using the commands we learned in [Exploration](/santiment-queries/exploration/).
+
+Here, we've provided the Bridge Transactions table along with its columns, concise descriptions, and a few sample queries for you to delve into the data.
+
+### bridge\_transactions
+
+The `bridge_transactions` table is designed to store information about liquidity pool creation events on DEX protocols. It has the following columns:
```

Comment:
In many cases liquidity pools are not involved in bridge project. I think we need to remove this

Reply:
Fixed, was a leftover i didn't notice